src/app/BUILD.gn

```diff
@@ -215,14 +220,27 @@ static_library("app") {
     sources += [
       "SimpleSubscriptionResumptionStorage.cpp",
       "SimpleSubscriptionResumptionStorage.h",
+      "SubscriptionResumptionSessionEstablisher.cpp",
```

Comment:
Previously adding ResumptionSessionEstablisher was added to source only if `chip_persist_subscriptions` is there a reason why that is being dropped?

Reply:
It is still the case; There were two ifs back to back checking the same thing so i merged them together.

Reply:
Thanks, `if (chip_persist_subscriptions) {` was above the fold